rubato/game.py

```diff
@@ -97,6 +97,8 @@ def constant_loop(cls):  # test: skip
         try:
             while True:
                 cls.update()
+        except KeyboardInterrupt:
+            sys.exit(0)
```

Comment:
Should also quit SDL. Use the quit code that's in the SDL quit event handling

Reply:
Cool beans